### ISSUE TITLE
[Security] PISS using DICSS

### ISSUE THREAD

**Issue by Operation420** [letsgetrandy/DICSS#55]
I have a system to Pass Information Securely to Server (PISS).

Is is possible to use DICSS to PISS?

**Comment by letsgetrandy** [comment]
Yes, this is possible, but you should take care to only pass the information to an external server. It is possible to PISS outward with DICSS, and you'll find this helps you to pool information. But those who attempt to PISS internally have often found their systems to become infected.

**Comment by Operation420** [comment]
Thanks. I PISS externally often and using DICSS to PISS is not good if DICSS is too hard to PISS.

**Comment by Operation420** [comment]
Also I prefer to PISS in a private environment.

**Comment by v6** [comment]
// , Be careful of key generation. It's best not to use a virtual environment, but bare metal, instead, for this.
